app/models/agents/website_agent.rb

```diff
@@ -353,6 +376,12 @@ def receive(incoming_events)
     end
 
     private
+    def consider_response_successful?(response)
+      response.success? || begin
+        consider_success = options["consider_http_error_success"]
+        consider_success.present? && consider_success.include?(response.status)
```

Comment:
To support strings too:

`(consider_success.include?(response.status.to_s) || consider_success.include?(response.status))`